check-docstyle.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+directories="src deployments tests perf_tests PoC_code"
```

Comment:
These directories don't exist in this repository, leave this field blank for now.